### PR TITLE
License setup

### PR DESCRIPTION
**Pull request by jfrosorio**
I setup the license, but I'm not sure about the next line:

`Copyright (c) 2018 vueTablePro`

Specifically, I'm not sure about the "vueTablePro" part. Do you think this is ok?

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 vueTableTable
```
Comment (miguellteixeira) [inline LICENSE]:
```suggestion
Copyright (c) 2018 José Osório, Francisco Pereira, Miguel Teixeira 
```

Comment (miguellteixeira) [reply LICENSE]: Add the copyright holders instead of using the project name

Comment (jfrosorio) [reply LICENSE]: I guess the fairest way to write this is sorting the names by alphabetical order, like so:
Francisco Pereira, José Osório, Miguel Teixeira

Comment (miguellteixeira) [reply LICENSE]: Agreed. :+1:

Comment (chicop) [reply LICENSE]: 👍

**Pull request merged**